src/components/ClusterTable.tsx

```diff
@@ -1,27 +1,98 @@
-type StatID = string;
-interface Stats {
+import SortableTable from "./SortableTable/SortableTable";
+
+export type StatID = string;
+export type BestID = string;
+export type HtmlString = string;
+export type ClusterID = string;
+
+export interface Stats {
   mean: number;
   std: number;
 }
 
+export interface Header {
+  key: string;
+  value: string;
+  sort: string | boolean;
+  type: string;
+}
+
 export interface Cluster {
   rank: number | "Unclustered";
   id: number | "-";
   size: number;
-  best: string[];
+  best: Record<BestID, HtmlString>;
   stats: Record<StatID, Stats>;
 }
 
-interface Props {
+export interface Props {
   stat_labels: Record<StatID, string>;
```

Comment:
These are not just labels for the stat, but for every column header.
Can it be renamed to `labels`, `column_header_labels`, `headers` or something?